testing/simapp/app.go

```diff
@@ -913,6 +957,14 @@ func BlockedAddresses() map[string]bool {
 func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino, key, tkey storetypes.StoreKey) paramskeeper.Keeper {
 	paramsKeeper := paramskeeper.NewKeeper(appCodec, legacyAmino, key, tkey)
 
+	paramsKeeper.Subspace(authtypes.ModuleName)
+	paramsKeeper.Subspace(banktypes.ModuleName)
+	paramsKeeper.Subspace(stakingtypes.ModuleName)
+	paramsKeeper.Subspace(minttypes.ModuleName)
+	paramsKeeper.Subspace(distrtypes.ModuleName)
+	paramsKeeper.Subspace(slashingtypes.ModuleName)
+	paramsKeeper.Subspace(govtypes.ModuleName)
+	paramsKeeper.Subspace(crisistypes.ModuleName)
```

Comment:
These modules have been migrated to use module maintained parameters, I don't think we need to register subspaces for them anymore.

Reply:
K, I'll make a PR to sdk to remove them there too.  (idea is to get the two app.go's basically matching)

Reply:
Sounds good to me! I could see the sdk guys possibly wanting to keep them around for the remainder of the `x/params` lifetime but maybe they'll be fine with removing them too! 

Appreciate the effort in getting both app configs matching pretty well, naturally there will be some divergence but standardising on boilerplate is good! I personally always liked the way osmosis split up their app config with `AppKeepers` etc. but never got around to refactoring simapp to use that pattern.